File: proyecto_java/Problema1/src/registroFutbol/registroFutbol.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package registroFutbol;
import java.util.Scanner;

/**
 *
 * @author ASUS
 */
public class registroFutbol {
    public static void registrarParticipanteFutbol() {
        Scanner entrada = new Scanner(System.in);
        String nombre;
        int edad;
        String posicion;
        String ciudad;
        String equipo;
        System.out.println("Ingrese el nombre del participante:");
        nombre = entrada.nextLine();
        System.out.println("Ingrese la edad del participante:");
        edad = entrada.nextInt();
        System.out.println("Ingrese la posicion del participante:");
        entrada.nextLine();
        posicion = entrada.nextLine();
        System.out.println("Ingrese la ciudad del participante:");
        ciudad = entrada.nextLine();
        System.out.println("Ingrese el equipo favorito del participante:");
        equipo = entrada.nextLine();
        System.out.printf("Datos\nNombre: %s\nEdad: %d\nPosicion: %s\nCiudad: %s\nEquipo: %s\n", nombre,
                edad, posicion, ciudad, equipo);

    }
    
}

File: proyecto_python/Procedimientos/registroFutbol.py
def registrarParticipanteFutbol():
    nombre = input("Ingrese el nombre del participante: ")
    edad = int(input("Ingrese la edad del participante: "))
    posicion = input("Ingrese la posicion en la que juega el participante:")
    ciudad = input("Ingrese la ciudad del participante: ")
    equipo = input("Ingrese el equipo del participante: ")


    cadena = (f"Datos:\nNombre: {nombre}\nEdad: {edad}\nPosicion: {posicion}\nCiudad: {ciudad}\nEquipo: {equipo}\n")

    return cadena